### ISSUE TITLE
Increase a11y of pagination

### ISSUE THREAD

**Issue by benaltair** [bahaistudies/jbs#58]
Mainly `pagination.tpl` will need to be updated. It does have some `aria-hidden` tags for the arrows which is nice, but this could be improved.
- [x] Test with screen reader and record pain points
- [x] Add attributes as needed
- [x] Adjust semantic HTML as needed

**Comment by benaltair** [comment]
Current HTML isn't bad at all.
```html
	<nav aria-label="View additional pages">
		<ul class="pagination justify-content-center">
			<li class="page-item disabled">
				<a class="page-link" href="">
					<span aria-hidden="true">←</span>
					Previous
				</a>
			</li>
			<li class="page-item active">
				<span class="page-link">
					1-48 of 71
				</span>
			</li>
			<li class="page-item">
				<a class="page-link" href="https://journal.bahaistudies.ca/online/issue/archive/2">
					Next
					<span aria-hidden="true">→</span>
				</a>
			</li>
		</ul>
	</nav>
```

**Comment by benaltair** [comment]
Pain points so far:
- Previous isn't disabled on the first page and vice versa for the last page. Seems like it's being disabled for mouse users, but not for keyboard.
- Instead of "next page" and "previous page" it should describe which page number. This should be somewhat implied for sighted users because of the item range displayed between. This is accessible for a screen reader but isn't intuitive I think when using the navigation.
